add a page with instructions to integrate jupyter extension with Goog…

jhl2015 opened the pull request:
…le Vertex AI

- added a page under cloud-guides with instructions on how to integrate Teradata Jupyter extensions with Google Vertex AI.
- uploaded 3 png files
- added the new page under 'How to' section in the left navigation bar

ltammariello commented on modules/cloud-guides/pages/integrate-teradata-jupyter-extensions-with-google-vertex-ai.adoc:
```diff
@@ -0,0 +1,162 @@
+ 
+# PIP install the Teradata Python & R libraries
+pip install teradataml
+Rscript -e "install.packages('tdplyr',repos=c('https://teradata-download.s3.amazonaws.com','https://cloud.r-project.org'))"
```
This line needs to change to point to the new Teradata R repo:

Rscript -e "install.packages('tdplyr',repos=c('https://r-repo.teradata.com','https://cloud.r-project.org'))"

ltammariello replied:
Also, as I'm thinking about it, if R and/or the R kernel is not installed in the Vertex AI environment this will not work.   Maybe not it as optional or leave it out.  Python and the Python kernel should always be there as part of Jupyter so leaving the teradataml is probably ok?

jhl2015 replied:
fixed repo. commented this out by default and left a comment to uncomment it if user uses an environment that supports R.

ltammariello commented on modules/cloud-guides/pages/integrate-teradata-jupyter-extensions-with-google-vertex-ai.adoc:
```diff
@@ -0,0 +1,162 @@
+# Copy notebooks
+COPY ./ShippedNotebooks/ /home/jupyter
+COPY ./StartHere.ipynb ./license.txt /home/jupyter
+ 
```
These notebooks aren't included in the Linux desktop build.  If they want to include the sample notebooks we include in the Linux desktop build those are in the "notebooks" directory of wherever they unzipped the download.

jhl2015 replied:
fixed.

jhl2015 replied:
fixed

ltammariello commented on modules/cloud-guides/pages/integrate-teradata-jupyter-extensions-with-google-vertex-ai.adoc:
```diff
@@ -0,0 +1,162 @@
+RUN pip install teradataml && \
+    pip install plotly && \
+    Rscript -e "install.packages('tdplyr',repos=c('https://teradata-download.s3.amazonaws.com','https://cloud.r-project.org'))"
+----
```
R repo change:

Rscript -e "install.packages('tdplyr',repos=c('https://r-repo.teradata.com','https://cloud.r-project.org'))"

ltammariello replied:
Same comment here about R...make optional or leave it out as it won't work if R and the R kernel are not installed.

jhl2015 replied:
fixed repo and add a comment to use it only if base image support R. as we are doing in the sample dockerfile, we use r-cpu base images.

ltammariello commented on modules/cloud-guides/pages/integrate-teradata-jupyter-extensions-with-google-vertex-ai.adoc:
```diff
@@ -0,0 +1,162 @@
+    jupyter labextension install --no-build teradata_sqlhighlighter* && \
+    jupyter labextension install --no-build teradata_connection_manager* && \
+    jupyter labextension install --no-build teradata_preferences* && \
+    jupyter labextension install --no-build jupyterlab-plotly && \
```
I would leave out the plotly extensions.  We include those in our docker but users don't need to include them.

jhl2015 replied:
removed plotly.

ltammariello commented on modules/cloud-guides/pages/integrate-teradata-jupyter-extensions-with-google-vertex-ai.adoc:
```diff
@@ -0,0 +1,162 @@
+# Teradata R tdplyr library will be installed from S3
+ 
+RUN pip install teradataml && \
+    pip install plotly && \
```
I would leave out the plotly install, see above.

jhl2015 replied:
removed plotly

adamtwo commented:
I've a couple or things:
1. Could you please move the images to a subdirectory that is named the same way as the .doc file? We have this convention to make it easier for us to find out what image is used where.
2. For the container-based solution, is there a reason why we don't publish this container to DockerHub?

ltammariello commented:
> I've a couple or things:
> 
> 1. Could you please move the images to a subdirectory that is named the same way as the .doc file? We have this convention to make it easier for us to find out what image is used where.
> 2. For the container-based solution, is there a reason why we don't publish this container to DockerHub?

Regarding publishing to DockerHub, I think we (dev. team) don't really want to be tracking/maintaining multiple docker images, different ones for different integration tools etc.

adamtwo commented:
ok, got it. @jhl2015 , the only thing left is to move the images to `integrate-teradata-jupyter-extensions-with-google-vertex-ai` subdir.

jhl2015 commented:
> ok, got it. @jhl2015 , the only thing left is to move the images to `integrate-teradata-jupyter-extensions-with-google-vertex-ai` subdir.

moved the 3 images to subdir and updated .adoc file to reflect the changes.
